modules/serverpod_auth/serverpod_auth_server/lib/src/generated/user_image.dart

```diff
@@ -76,7 +76,7 @@ abstract class UserImage extends _i1.TableRow {
   @Deprecated('Will be removed in 2.0.0')
   Map<String, dynamic> toJsonForDatabase() {
     return {
-      if (id != null) 'id': id,
+      'id': id,
```

Comment:
Because I reverted back to the original implementation. I suppose we could still have the if here but then I need to do a special implementation for this method. Considering that we are not using it and that it is deprecated I think it is okay to freeze the functionality in time here.

Reply:
Note: The referenced "original implementation" is not what is in main currently but what it was before the null checks where introduces and the method was marked as deprecated.

We skip adding the additional functionality since it is going to be removed anyway.